docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -24,10 +32,41 @@ To create a new user:
 4. Review the new user information and note the temporary password that has been assigned. To send the user an introductory email, select the checkbox.
 5. Press *Done* when complete.
 
+## Users Table
+
+The *Users* table displays a list of users who are authorized to login to the Run:ai platform.
+
+!!! Note
+    SSO users who have not logged into the Run:ai platform will not appear in the table until they do.
+
+The *Users* table contains informtion about users who have logged into the Run:ai Platform. The table includes columns for:
+
+* **User**&mdash;the email and user ID of the user.
+* **Type**&mdash;the type of user (local or SSO).
+* **Last login**&mdash;the last time the user logged into the platform.
+* **Access rule(s)**&mdash;press *VIEW* to view the access rule(s) assigned to the user.
+* **Created by**&mdash;indicates who created the user in the able.
+* **Creation time**&mdash;the timestamp for when the user was created.
+* **Last updated**&mdash;the last time the user information was updated.
+
+When you press *View* in the *Access rule(s)* column, a pop-up will appear that displays the rules assigned to the user. In the popup are the following columns:
+
+* **Role**&mdash;the name of the role assigned to the user.
+* **Scope**&mdash;the scope to which the user has rights.
+* **Type**&mdash;the type of user (SSO or other)
+* **Group**&mdash;the group to which the user is assigned. (Applies only to SSO users)
+* **Athorized by**&mdash;the user who granted the access roles.
+* **Creation time**&mdash;the timestamp for when the user was created.
+* **Last updated**&mdash;the last time the user information was updated.
+
 ## Assigning access rules to users
 
 Once you have created the users you can assign them *Access rules*. This provides the needed authorization to access system assets and resources.
 
+### Roles and permissions
+
+Roles provide a way for administrators to group and identify collections of permissions that administrators assign to [subjects](../runai-setup/access-control/rbac.md#subjects). Permissions define the actions that can be performed on managed entities. The [Roles](../runai-setup/access-control/rbac.md#roles) table shows the default roles and permissions that come with the syste. See [Role based access control](../runai-setup/access-control/rbac.md) for more information.
+
 To add an *Access rule* to a user:
```

Comment:
@jasonnovichRunAI 
Maybe we should also mention that Access rules can be added via he access rules page? because if an SSO user still didn't login, the admin won't see it in the Users page, but they can still add access rules to these users

Reply:
@jasonnovichRunAI ping on my last comment

Also, maybe we should write "To add or edit an Access rule" to a user:" instead of just "add"